dracut-initramfs-restore.sh

```diff
@@ -10,16 +10,24 @@ KERNEL_VERSION="$(uname -r)"
 
 [[ $dracutbasedir ]] || dracutbasedir=/usr/lib/dracut
 SKIP="$dracutbasedir/skipcpio"
+
 [[ -x $SKIP ]] || SKIP=cat
 
 [[ -f /etc/machine-id ]] && read MACHINE_ID < /etc/machine-id
 
 mount -o ro /boot &>/dev/null || true
 
-if [[ $MACHINE_ID ]] && [[ -d /boot/${MACHINE_ID} || -L /boot/${MACHINE_ID} ]] ; then
-    IMG="/boot/${MACHINE_ID}/${KERNEL_VERSION}/initrd"
+if [[ -d /efi/loader/entries || -L /efi/loader/entries ]] \
+    && [[ $MACHINE_ID ]] \
+    && [[ -d /efi/${MACHINE_ID} || -L /efi/${MACHINE_ID} ]] ; then
+        IMG="/efi/${MACHINE_ID}/${KERNEL_VERSION}/initrd"
+    elif [[ -d /boot/loader/entries || -L /boot/loader/entries ]] \
+        && [[ $MACHINE_ID ]] \
+        && [[ -d /boot/${MACHINE_ID} || -L /boot/${MACHINE_ID} ]] ; then
+        IMG="/boot/${MACHINE_ID}/${KERNEL_VERSION}/initrd"
+    else
```

Comment:
please indent to start of line (same as `if` and `fi`) to be consistent with the rest of the code style